.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Publish Docker image
+
+on:
+  push:
+    # Push  to `master` or `development`
+    branches:
+      - master
+      - development
+      - add_docker-publish
```

Comment:
I'd also add `workflow_dispatch:` as a possible option to `on:`. It let's you manually run the workflow.

https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch

Reply:
Done